src/core/Config.jl

```diff
@@ -22,13 +22,14 @@ function default_config()
                 :basename => "data_one",
                 :io_backend => "HDF5",
                 :nsteps => 10,
+                :step_conversion_factor => 10,
```

Comment:
sorry I might have missed it from our conversations, what's the `step_conversion_factor`?